### PR TITLE
deployment: add chapter

### PR DESCRIPTION
**Pull request by idg-sam**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: 4-Deployment/deploy-to-azure-app-service/README.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+- [Visual Studio Code](https://code.visualstudio.com/download) is recommended for running and editing this sample.
+- [VS Code Azure Tools Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-node-azure-pack) extension is recommended for interacting with **Azure** through VS Code interface.
+- An **Azure subscription**. This sample uses the free tier of **Azure App Service**.
+
+- [Maven 3.6](https://maven.apache.org/download.cgi) or higher installed and set up in path environment (i.e., callable via `mvn` command)
+- [Maven extension for VSCode](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-maven)
+
+## Setup
+
+Follow the setup instructions in [Enable your Java Spring Boot web app to sign in users and call Microsoft Graph with the Microsoft identity platform](https://github.com/azure-samples/ms-identity-java-spring-tutorial/2-Authorization-I/call-graph) sample.
```
Comment (Shama-K) [inline 4-Deployment/deploy-to-azure-app-service/README.md:36]: Path is not correct.

File: 4-Deployment/deploy-to-azure-app-service/README.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
+# Deploy your Java applications to Azure Cloud and use Azure App Service to manage your operations
```
Comment (kalyankrishna1) [inline 4-Deployment/deploy-to-azure-app-service/README.md:1]: >applications [](http://example.com/codeflow?start=19&length=12)

link this readme in the main landing page

File: 4-Deployment/deploy-to-azure-app-service/README.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+3. Prepare your web app for deployment.
+4. Deploy to Azure App Service.
+
+### Step 1: Create a new app on Azure App Service
+
+1. Open the VSCode command palette (ctrl+shift+P on Windows and command+shift+P on Mac).
+1. Choose  `Azure App Service: Create New Web App...`
+1. Enter a globally unique name for your web app (e.g. `example-domain`) and press enter. Make a note of this name. If you chose `example-domain` for your app name, your app's domain name will be `https://example-domain.azurewebsites.net`
+1. Select `Java 11` for your runtime stack.
+1. Select `Java SE (Embedded Web Server)` for your Java web server stack.
```
Comment (kalyankrishna1) [inline 4-Deployment/deploy-to-azure-app-service/README.md:62]: Step for `OS` is missing

**Comment by kalyankrishna1** [comment]
#approved

**Review by Shama-K — APPROVED** [review]
LGTM.
Thanks Sam.

File: README.md [hunk 4]
```diff
@@ -64,11 +64,11 @@ Please refer to each sample's README for sample-specific recommendations.
...
 
-<!-- ### Chapter 4: Deploy your app to Azure
+### Chapter 4: Deploy your app to Azure
 
 |                 |               |
 |-----------------|---------------|
-| <img src="./ReadmeFiles/sign-in.png" width="200"> | [**4.1 Deploy to Azure App Service**](./4-Deployment/deploy-to-azure-app-service) </br> Prepare your app for deployment to Azure App Service. Learn how to package and upload files, configure authentication parameters and use various Azure services for managing your operations. | -->
+| <img src="./ReadmeFiles/sign-in.png" width="200"> | [**4.1 Deploy to Azure App Service**](./4-Deployment/deploy-to-azure-app-service) </br> Prepare your app for deployment to Azure App Service. Learn how to package and upload files, configure authentication parameters and use various Azure services for managing your operations. |
```
Comment (derisen) [inline README.md:71]:
```suggestion
| <img src="./ReadmeFiles/sign-in.png" width="200"> | [**4.1 Deploy to Azure App Service**](./4-Deployment/deploy-to-azure-app-service) </br> Prepare your app for deployment to Azure App Service. Learn how to package and upload files, configure authentication parameters and use various Azure services for managing your operations. |
```

Might be better to link to the README itself i.e. `./4-Deployment/deploy-to-azure-app-service/README.md`

**Review by derisen — APPROVED** [review]
Clear instructions, deployment works fine -LGTM!

**Pull request merged**